internal/dependency/health_service.go

```diff
@@ -114,16 +113,10 @@ func (d *HealthServiceQuery) Fetch(clients dep.Clients) (interface{}, *dep.Respo
 		Near:       d.near,
 	})
 
-	u := &url.URL{
-		Path:     "/v1/health/service/" + d.name,
-		RawQuery: opts.String(),
-	}
-	if d.tag != "" {
-		q := u.Query()
-		q.Set("tag", d.tag)
-		u.RawQuery = q.Encode()
-	}
-	//log.Printf("[TRACE] %s: GET %s", d, u)
+	//log.Printf("[TRACE] %s: GET %s", d, &url.URL{
```

Comment:
Thanks for keeping the commented out log up to date. It will be helpful when I get a chance to start reviewing them for public consumption.

Reply:
Yeah np! It threw me off for a bit what the variables `u` and `q` were used for so i thought to help future self i'd just remove it.